src/Microsoft.Framework.Runtime/FileGlobbing/MatcherExtensions.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Microsoft.Framework.FileSystemGlobbing;
+using Microsoft.Framework.FileSystemGlobbing.Abstractions;
+
+namespace Microsoft.Framework.Runtime.FileGlobbing
```

Comment:
Keep the namespace the same as globbing. `Microsoft.Framework.FileSystemGlobbing`